src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -883,8 +883,8 @@ namespace CommandLineInterface {
             if (!ofs.good()) {
                 ShowFatalError(state, "EnergyPlus: Could not open file \"" + RVIfile.string() + "\" for output (write).");
             } else {
-                ofs << state.files.eso.filePath << '\n';
-                ofs << state.files.csv.filePath << '\n';
+                ofs << state.files.eso.filePath.string() << '\n';
```

Comment:
yes. ostream operator<< with a filesystem path will double quote the path. Moving to the string overload alleviates the problem.